mycore-classeditor/src/main/resources/components/classeditor/config/mycore.properties

```diff
@@ -2,3 +2,5 @@ MCR.Jersey.Resource.Packages=%MCR.Jersey.Resource.Packages%,org.mycore.frontend.
 MCR.LayoutTransformerFactory.Default.Ignore=%MCR.LayoutTransformerFactory.Default.Ignore%,classificationEditor
 
 MCR.Module-solr.linkQuery=category.top:{0}
+
+MCR.classeditor.xlanguages=disable,display,group,uri
```

Comment:
In my opinion it would be easier to keep the "x-" prefix in the property:
`MCR.classeditor.xlanguages=x-disable,x-display,x-group,x-uri`

I do not remember any usecase where you need the language id without the prefix "x-"?

Reply:
> I do not remember any usecase where you need the language id without the prefix "x-"?

That's my thinking as well. Because all relevant languages always have the `x-`-prefix, it is unnecessary to include this prefix in every configured value. Omitting them, saves some keystrokes.

Should the property than be called `MCR.classeditor.languages` instead of `MCR.classeditor.xlanguages`? Because withot the automatic inclusion, it would be possible to add languages without the `x-`-prefix; say `sgn` (Sign Languages), ...

Reply:
I like "additionalLanguages", as you said, that gives us the opportunity to add "our" x-languages" and possible other languages without the "x"- prefix.